package.json

```diff
@@ -77,6 +82,7 @@
     "generate-alpharelease": "eslint-generate-prerelease alpha",
     "generate-betarelease": "eslint-generate-prerelease beta",
     "generate-rcrelease": "eslint-generate-prerelease rc",
-    "publish-release": "eslint-publish-release"
+    "publish-release": "eslint-publish-release",
+    "prepare": "node node_modules/yorkie/bin/install.js"
```

Comment:
Adding a `prepare` script doesn't seem necessary because `yorkie` already runs its `bin/install.js` on [install](https://github.com/yyx990803/yorkie/blob/a4cf01d789da2633a33a888b496fa35395e72109/package.json#L11).

Reply:
Sure, just removed it